nano/node/bootstrap/bootstrap_service.cpp

```diff
@@ -204,8 +206,24 @@ bool nano::bootstrap_service::send (std::shared_ptr<nano::transport::channel> co
 
 	// TODO: There is no feedback mechanism if bandwidth limiter starts dropping our requests
```

Comment:
Is this now handled by the added callback ? Or is the TODO about something else ?

Reply:
This todo is now out of date